.github/workflows/_deploy_production.yml

```diff
@@ -143,6 +143,43 @@ jobs:
           comment:
             "Apply Run from GitHub Actions CI ${{ inputs.tag }}"
 
+  # Some renegade users are self-hosting these, so support them as best we can by making our
+  # infrastructure images available to them.
+  publish-infra-images:
+    # Only publish if our own deploy was successful
+    needs: deploy-production
+    runs-on: ubuntu-22.04
+    permissions:
+      packages: write
+      id-token: write
+    strategy:
+      fail-fast: false
+      matrix:
+        image: [domain, api, web, relay]
```

Comment:
Do we deploy all of these at the same time always? I thought we also sometimes deploy hotfixes that only affect some of these components?

Reply:
The deploy workflow and the hotfix are the same workflow now because we don't trigger infra deploys on component releases. Yeah we always build and deploy all news images on each deploy.